scripts/src/java/org/oppia/android/scripts/maven/GenerateMavenDependenciesList.kt

```diff
@@ -214,11 +217,11 @@ class GenerateMavenDependenciesList(
   }
 
   /**
-   * Helper function to parse the textproto file to a proto class.
+   * Helper function to parse the textProto file to a proto class.
```

Comment:
```suggestion
   * Helper function to parse the text proto file to a proto class.
```

In this case since it's documentation.

Reply:
Done.